app-template/decred/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
+  "project_info": {
+    "project_number": "31679606073",
+    "firebase_url": "https://decred-wallet.firebaseio.com",
+    "project_id": "decred-wallet",
+    "storage_bucket": "decred-wallet.appspot.com"
```

Comment:
These point to the Firebase account that's used for testing the push notifications in the mobile app suite – we'll need to create a production account and change the URLs here.